reference/features/authentication.md

```diff
@@ -1,41 +1,45 @@
 # Authentication
 
-MeiliSearch uses a key-based authentication. There are three types of keys:
+MeiliSearch uses key-based authentication.
 
-- The **Master** key grants access to all routes.
-- The **Private** key grants access to all routes except the `/keys` routes.
-- The **Public** key only grants access to the following routes:
-  - `GET /indexes/:index_uid/search`
-  - `POST /indexes/:index_uid/search`
-  - `GET /indexes/:index_uid/documents`
-  - `GET /indexes/:index_uid/documents/:doc_id`
-- Without any key, you can always access `GET /health`
+When using MeiliSearch in `production`, authentication is mandatory. You can read more about `production` mode in the [configuration reference](/reference/features/configuration).
+
+If a MeiliSearch instance does not use authentication, all routes will be publicly accessible and unprotected.
+
+## Key types
 
-When a master key is provided to MeiliSearch, both the private and the public keys are automatically generated. **You cannot create any additional keys**.
+MeiliSearch uses three types of keys:
+
+- The **Master** key grants access to all routes
+- The **Private** key grants access to all routes except the `/keys` routes
+- The **Public** key only grants access to the following routes:
+  - [`GET /indexes/:index_uid/search`](/reference/api/search#search-in-an-index-with-get-route)
+  - [`POST /indexes/:index_uid/search`](/reference/api/search#search-in-an-index-with-post-route)
+  - [`GET /indexes/:index_uid/documents`](/reference/api/documents#get-documents)
+  - [`GET /indexes/:index_uid/documents/:document_id`](/reference/api/documents#get-one-document)
 
-## Master Key
+When a master key is provided, both the private and the public keys are automatically generated. **You cannot create any additional keys**.
 
-When launching an instance, you have the option of giving a master key. By doing so, all routes will be protected and will require a key to be accessed.
+The only unprotected route available when using the built-in key-based authorization is [`GET /health`](/reference/api/health).
 
-You can specify it by passing the `MEILI_MASTER_KEY` environment variable, or using the command line argument `--master-key`.
+## Adding a key
 
-You can retrieve both the private and the public keys using the master key on the [keys route](/reference/api/keys.md).
+When launching a MeiliSearch instance, you have the option of giving it a master key. By doing so, all routes will be protected and will require a key to be accessed.
 
-#### No master key
+You can specify a key by passing the `MEILI_MASTER_KEY` environment variable, or using the command line argument `--master-key`.
```

Comment:
````suggestion
You can specify the master key by setting the `MEILI_MASTER_KEY` environment variable, or passing the command line argument `--master-key` on launch.

#### Example

:::: tabs

::: tab Env

```bash
export MEILI_MASTER_KEY=whatever_you_want
meilisearch
```

:::

::: tab CLI

```bash
meilisearch --master-key=whatever_you_want
```

:::

::::
````
Not 100% sure I typed that right, but should create an example code block with two tabs, one for env and one for CLI. If you go with this suggestion, verify that it displays correctly before merging :)

Reply:
Could also ignore this for now and add it in a separate PR if preferred.

Reply:
sounds great, but one minor comment: what do you say we replace `whatever_you_want` with something like `[YOUR_MASTER_KEY]`? Sure, people could use whatever string they want, but generally we'd like them to use something that's hard to guess and brute-force, no?

Reply:
Yes, great suggestion :)